src/desktop/space/output.rs

```diff
@@ -39,7 +39,7 @@ pub struct OutputState {
 
     // damage and last_state are in space coordinate space
     pub old_damage: VecDeque<Vec<Rectangle<i32, Physical>>>,
-    pub last_toplevel_state: IndexMap<ToplevelId, Rectangle<i32, Physical>>,
+    pub last_toplevel_state: IndexMap<ToplevelId, (usize, Rectangle<i32, Physical>)>,
```

Comment:
At this point we might want to add a doc comment explaining the contents of this map.

Reply:
done